src/proxy/app/routes/request_manager.py

```diff
@@ -86,6 +87,10 @@ async def process_request(
         deployment = await self.config.get_catalog_by_deployment_name(authorize_response)
         response, http_status_code = await call_method(model, deployment)
 
+        if not isinstance(response, AsyncGenerator) and "model" in response:
+            response = self.model_to_dict(response)
+            response["model"] = response["model"] + ":" + deployment.location.lower()
```

Comment:
Why not just pop that as a new property in the response?

Reply:
I tried that, but the via te SDKs there is no access to the additional the property - this provides an easy work around...